lib/validate-license-key.js

```diff
@@ -0,0 +1,18 @@
+/*
+ * Copyright 2025 New Relic Corporation. All rights reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+'use strict'
+
+/**
+ * Verifies that a license key conforms to the license key spec. That is,
+ * the key is comprised of ASCII characters, excluding space (0x20), with a
+ * length between 10 and 64.
+ *
+ * @param {string} key The key to validate
+ * @returns {boolean} True if it is a valid key
+ */
+module.exports = function validateLicenseKey(key) {
```

Comment:
as i mentioned in the place this is used. This is not the intent of the invalid license key code, we're relying on the response from collector to determine this. I know you went through and updated all the fake keys to something that conforms but you can undo that as well